### PR TITLE
Remove chomp from internal formatter and add chomp_record option

### PR DESCRIPTION
**Pull request by simukappu**
*Issue #, if available:*
#142 #144 

*Description of changes:*
Remove chomp from internal formatter (merge #144) and add chomp_record option for compatible format with plugin v2.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -289,6 +289,10 @@ Specifing compression way for data of each record. Current accepted options are
 ### log_truncate_max_size
 Integer, default 1024. When emitting the log entry, the message will be truncated by this size to avoid infinite loop when the log is also sent to Kinesis. The value 0 means no truncation.
 
+### chomp_record
+Boolean. Default `false`. If it is enabled, formatter calls chomp and removes separator from the end of each record. This option is for compatible format with plugin v2.
+See [#142](https://github.com/awslabs/aws-fluent-plugin-kinesis/issues/142) for more details.
+
```
Comment (riywo) [inline README.md]: Could you add more details for firehose?

File: README.md [hunk 2]
```diff
@@ -351,7 +355,7 @@ Here are `kinesis_firehose` specific configurations.
 Name of the delivery stream to put data.
 
 ### append_new_line
-Boolean. Default `true`. If it is enabled, the plugin add new line character (`\n`) to each serialized record.
+Boolean. Default `true`. If it is enabled, the plugin add new line character (`\n`) to each serialized record. At first, the overridden formatter in the plugin calls chomp and removes separator from the end of each record. Then, it appends `\n` to each record.
```
Comment (riywo) [inline README.md]: It is unclear how it works with `chomp_record`. Could you add more details?

File: README.md (hunk 1 above)
Comment (simukappu) [reply README.md]: Thanks, updated like [this](https://github.com/awslabs/aws-fluent-plugin-kinesis/pull/172/files#diff-04c6e90faac2675aa89e2176d2eec7d8R292).

File: README.md (hunk 2 above)
Comment (simukappu) [reply README.md]: Thanks, updated like [this](https://github.com/awslabs/aws-fluent-plugin-kinesis/pull/172/files#diff-04c6e90faac2675aa89e2176d2eec7d8R357).

**Comment by riywo** [comment]
LGTM

**Pull request merged**